requester/requester.go

```diff
@@ -26,6 +27,7 @@ import (
 	"os/signal"
 	"strings"
```

Comment:
What about just consuming b.results as more requests are in flight and outputing the number of completed requests once a while? We can avoid having a progressCounter.